README.md

```diff
@@ -2,19 +2,40 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+```
+class Human {
+  constructor(name, age){
+    this.name = name;
+    this.age = age;
+  }
+
+  ageOneYear(){
+  this.age++
+  return `One year has passed now I'm ${this.age}`
+  }
+
+  eating(){
+    return `Just ate. mmm, mmm, mmm, I'm love'n it.`
+  }
+}
+
+let thomas = new Human('Thomas', 30);
+console.log(thomas.ageOneYear());
+
```

Comment:
Nitpick, but make sure you do everything the assignment ask of you:
    console log your humans age
    * call ageOneYear on your human
    * console log their age again.
    * call eating on your human.